### ISSUE TITLE
Privacy Policy unavailable without internet connection

### ISSUE THREAD

**Issue by brs17** [pop-os/iso#143]
If this is a bug, please use the template below. If this is a feature request, question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os - as that is the proper forum for those types of posts.

**Reporter info**

```
 * Distribution - (run ```cat /etc/os-release```)
 * Related Application and/or Package Version - apt policy $PACKAGE NAME
```
NAME="Pop!_OS"
VERSION="17.10 (Artful Aardvark)"
BUILD = 31 (noNVIDIA)

**Issue/Bug Description**
If Pop!_OS installation is done without an Internet connection, the privacy policy for the Mozilla Location Service returns an error: "Error resolving " location.services.mozilla.com": Name or service not known"

**Steps to reproduce (if you know)**
Install Pop!_OS without an Internet connection.

**Expected behaviour**

This is up for debate. Some ideas:
HTTP link should be provided to view the policy at a later time or not shown when an Internet connection is unavailable, but shown when first used (after installation).

**Other Notes**

**Comment by isantop** [comment]
I wonder if it wouldn't be better to just provide this document with the ISO for offline viewing. It probably wouldn't be too hard to have it try to get the latest version from the website, but if it fails fall back on our cached copy, with a disclaimer that it may be out of date and the latest version is "always available online at https://foo.com/bar"

**Comment by brs17** [comment]
I think that works, or just " the latest version is available online at https://foo.com/bar". It's not like one can use the location services without Internet access.

**Comment by isantop** [comment]
That's true, but it's still a good idea to let people know where all of the documentation is. It makes me uneasy shipping code that deal with privacy that we can't inform the user about, even if that code isn't active. What happens if someone sets up their computer without internet, then later connects and enables location services? Do we need to track that they haven't seen the privacy policy and show it to them retroactively?

**Comment by brs17** [comment]
@isantop So I just ran an install without an Internet connection. Then after completing the install, I connected to the internet and went into the Privacy Settings to enable Location Services. It is just a toggle and there is no mention of the Privacy Policy. 

For what it is worth however, when launching Firefox for the first time, a tab does open to this page: [https://www.mozilla.org/en-US/privacy/firefox/](https://www.mozilla.org/en-US/privacy/firefox/). I would argue this is not sufficient.
